website/docs/About/index.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: About
+sidebar_position: 1
+---
+
+# About the IDAES Flowsheet Processor
+
```

Comment:
Obviously add some content here later. Would suggest a list of functionality and a screenshot. May also be useful to have a "System requirements" section (e.g., no old macs, Linux/Unix is just a maybe, etc.)